Data/Scripts/GardenConquest/Extensions/GridExtensions.cs

```diff
@@ -184,6 +186,48 @@ public static class GridExtensions {
 			return false;
 		}
 
+		/// <summary>
+		/// Get the classifier block on the grid
+		/// </summary>
+		/// <param name="grid"></param>
+		/// <remarks>Grids should only have 1 classifier block</remarks>
+		/// <returns>The HullClassifier as IMyCubeBlock if found, null otherwise</returns>
+		public static IMyCubeBlock getClassifierBlock(this IMyCubeGrid grid) {
+			List<IMySlimBlock> beaconBlocks = new List<IMySlimBlock>();
+
+			// Get all beacon blocks
+			Func<IMySlimBlock, bool> isBeacon = b => b.FatBlock != null && b.FatBlock is InGame.IMyBeacon;
+			grid.GetBlocks(beaconBlocks, isBeacon);
+
+			foreach (IMySlimBlock block in beaconBlocks) {
+				String subTypeName = block.FatBlock.BlockDefinition.SubtypeName;
+				if (subTypeName.Contains(Blocks.HullClassifier.SHARED_SUBTYPE)) {
+					return block.FatBlock;
+				}
```

Comment:
There's ```isClassifierBlock(IMySlimBlock block)``` in HullClassifier, should we just use that or  move it to a new class ```IMySlimBlockExtensions.cs```?